src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/AbstractSyntaxClassifier.cs

```diff
@@ -18,11 +18,11 @@ protected AbstractSyntaxClassifier()
         protected string GetClassificationForType(ITypeSymbol type)
             => type.GetClassification();
 
-        public virtual void AddClassifications(SyntaxNode syntax, SemanticModel semanticModel, ArrayBuilder<ClassifiedSpan> result, CancellationToken cancellationToken)
+        public virtual void AddClassifications(Workspace workspace, SyntaxNode syntax, SemanticModel semanticModel, ArrayBuilder<ClassifiedSpan> result, CancellationToken cancellationToken)
```

Comment:
added because we want to be able to pass the classifier all the way to the IEmbeddedLanguage system.  it will need to check user options to validate if it should actually classify or not.